docker/k8s-config/agent.json

```diff
@@ -10,4 +10,11 @@
   // metrics on the container running the agent.
   implicit_metric_monitor: false,
   implicit_agent_process_metrics_monitor: false,
+
+  // Checkpoints file is persisted across pod restarts to ensure we don't
+  // emit duplicate lines for container logs on on pod restart, but we don't
+  // want to persist checkpoint data for ephemeral log files (aka log files
+  // which are not persisted across restarts and are ephemeral to the
+  // container - think anything in /var/log/scalyr-agent-2/*.log)
+  ignore_checkpoints_on_startup_path_globs: ["/var/log/scalyr-agent-2/*.log"],
```

Comment:
I guess there shouldn't be so many reasons for users to deal with k8s agent config file directly, right? I just thought about putting that addition on the code level for the k8s monitor, so we can keep the config as "blank" as possible as we have for the "regular" agent config.

Reply:
Correct - on K8s (docker images), user shouldn't be editing base config files directly - they should either utilize environment variables or a ConfigMap to configure the agent.

And the problem is that this change and config option is global and not specific to k8s monitor (so putting config option into k8s monitor may make it a bit confusing).

But yeah, in short, I also don't think there should ever be a need to utilize this config option outside of the k8s environment unless a user has some specific use case.